### DISCUSSION TITLE
Extracting data using premium features like bio_location and profile Country

### DISCUSSION THREAD

**Discussion by prathamesh168** [DocNow/twarc#691] [Q&A] [answered]
anybody know how to extract tweets by using users location of home. We have premium access of twitter api .
Thank you

**Comment by igorbrigadir** [comment, marked as answer]
Note: the v1.1 Premium API was meant to be deprecated Feb 13th once the new Paid API plans are implemented, it may stop working suddenly. Premium API is implemented in `twarc` commands for v1.1.

However, if you are talking about the v2 API, and have Academic Access, you can only use `bio_location` as an operator in a Filter stream rule https://developer.twitter.com/en/docs/twitter-api/tweets/filtered-stream/integrate/build-a-rule#list , not in search. Also note that this matches on Text only, it is not a geo location. The User Location field is a free text field that can contain anything. In twarc, this operator would be used for twarc2 stream-rules commands: https://twarc-project.readthedocs.io/en/latest/twarc2_en_us/#stream
